### PR TITLE
Fix support for creating release posts

### PR DESCRIPTION
**Pull request by asarium**
This should have all the changes necessary for creating posts about the
releases on the forums.

### CODE REVIEW

File: config.yml.sample [hunk 1]
```diff
@@ -32,7 +32,11 @@ hlp:
...
-    hlp_board: 92
+    hlp_board: 173
+
+release:
+    hlp_board: 50
 
 templates:
-    nightly: templates/nightly.mako
\ No newline at end of file
+    nightly: /media/diverses/Code/Sources/linux/asarium/nightlybuild/templates/nightly.mako
```
Comment (chief1983) [inline config.yml.sample]: Will these lines have to be changed to full paths going forward or will leaving them as relative paths in the used config be ok?

Comment (asarium) [reply config.yml.sample]: Ah, damn. I used the wrong path in the sample. A relative path should be ok if the working directory is set correctly.

File: templates/release.mako [hunk 2]
```diff
@@ -52,23 +72,16 @@ You can use freely available tools like [url=http://www.cpuid.com/softwares/cpu-
...
-[size=8pt]Compiled on Ubuntu 14.04.4 LTS 32-bit, GCC 4.8.4[/size]
-The Ubuntu builds are built and provided as a reference, and also because Ubuntu is a very common Desktop Linux distribution.  We usually recommend Linux users compile their own builds, but if you happen to be running 32-bit Ubuntu, or 64-bit Ubuntu with 32-bit libraries installed, these may work fine for you.  There have been reports of package managers maintaining FSO packages in various Linux distributions, if you plan to package an FSO release for a distribution please let use know and we will include a reference to it in our release posts.
+${build(groups["MacOSX"].mainFile)}
 
-[url=http://scp.fsmods.net/builds/LINUX/fs2_open_3.7.4_RC2.tar.bz2]fs2_open_3.7.4_RC2.tar.bz2[/url] ([url=http://scp.indiegames.us/builds/LINUX/fs2_open_3.7.4_RC2.tar.bz2]Mirror[/url]) ([color=yellow][url=http://scp.fsmods.net/builds/LINUX/fs2_open_3.7.4_RC2.md5]MD5[/url][/color])
+[img]http://scp.indiegames.us/img/linux-icon.png[/img] [color=green][size=12pt]Linux 64-bit[/size][/color]
+[size=8pt]Compiled on Ubuntu 14.04.4 LTS 64-bit, GCC 5[/size]
```
Comment (chief1983) [inline templates/release.mako:78]: This likely doesn't apply any more, should probably be updated to whatever Travis uses, or just  removed.

Comment (asarium) [reply templates/release.mako:78]: That should match what Travis is using. I made sure to keep that part valid when I wrote the template.

**Review by chief1983 — APPROVED** [review]

**Pull request merged**